common/app/utils/form.js

```diff
@@ -65,6 +65,10 @@ export function getValidationState(field) {
     return null;
   }
 
+  if ((/http(s):\/\/glitch\.com\/edit\/#!\/.*/g).exec(field.value)) {
```

Comment:
`RegEx().teat()` would be more suitable here. We don't care much for matching parts, we just want to know if it matches or not.